xyo-android-library/src/main/java/network/xyo/sdk/bluetooth/advertiser/XyoBluetoothAdvertiser.kt

```diff
@@ -67,7 +67,7 @@ class XyoBluetoothAdvertiser(
         advertiser.advertisingData = advertiseData
         advertiser.advertisingResponse = responseData
         advertiser.changeContactable(true)
-        advertiser.changeAdvertisingMode(AdvertiseSettings.ADVERTISE_MODE_BALANCED)
+        advertiser.changeAdvertisingMode(AdvertiseSettings.ADVERTISE_MODE_LOW_LATENCY)
```

Comment:
This may be a naive comment on my part, but would this affect battery life on device? Could we still advertise in low power mode? Did you notice an increase in bound witness performance when changing this setting?

Reply:
I've not noticed degradation on battery life, but yes, this increased successfully bound witness with lower bw time gap.